index.ts

```diff
@@ -258,35 +270,38 @@ export class String {
     }
 }
 
+
+export class String extends $String { }
+
 export class StringBuilder {
     public Values: string[];
 
-    constructor(value?: string) {
+    constructor(value = '') {
         this.Values = [];
 
-        if (!String.IsNullOrWhiteSpace(value)) {
+        if (!$String.isNullOrWhiteSpace(value)) {
             this.Values = new Array(value);
         }
     }
 
     public ToString() {
```

Comment:
The methods of `String` were all changed to lowercase, why isn't that the case with the members of `StringBuilder`?

Reply:
i reverted them accidently 😄